domains/akimbo.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "Porfolio website for Akimbo,and Pixel Peak Studio",
+  "repo": "https://github.com/aksh120",
+  "owner": {
+    "username": "aksh120",
+    "email": "projects.akimbo@gmail.com"
+  },
+  "record": {
+    "NS": [
+      "casey.ns.cloudflare.com",
+      "rosalie.ns.cloudflare.com"
```

Comment:
Please give a reason as to why you want NS records.

Reply:
To link it with cloudflare.

Reply:
Not a valid reason.

Reply:
This ns leads to cloudflare once done I will connect it to my plesk server

Reply:
I don't think thats a valid reason still.

Reply:
So what do I do? To connect it with cloudflare

Reply:
You need a valid reason for it, and as far as I've seen, you don't have one.

Reply:
Reason why I need a valid reason is so that it doesn't get used for malicious stuff.

Reply:
What reason can I give I just want to make a portfolio website of mine